### PR TITLE
Fixing multiple golint issues

### PR DESCRIPTION
**Pull request by Pensu**
Signed-off-by: Peeyush Gupta <peeyushgupta91@gmail.com>

As suggested by @prateekpandey14, merged #660 #663 #664 #659

### CODE REVIEW

**Comment by prateekpandey14** [comment]
@Pensu merge #657 too

**Comment by Pensu** [comment]
@sonasingh46 Have made the changes, also made the changes asked for in another PRs as well.

@prateekpandey14 Have merged all the open PRs in this one.

File: cmd/maya-apiserver/spc-watcher/storagepool_delete.go [hunk 1]
```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/openebs/maya/pkg/storagepool"
 )
 
+
```
Comment (princerachit) [inline cmd/maya-apiserver/spc-watcher/storagepool_delete.go]: Remove this line.

**Comment by Pensu** [comment]
@princerachit Done!

**Pull request merged**